### DISCUSSION TITLE
After adding an alert rule, no alert event is generated in the history

### DISCUSSION THREAD

**Discussion by ghost** [ccfos/nightingale#1179] [Q&A]
夜莺前端版本：5.12.0
夜莺后端版本：5.8.0
使用告警规则中PromQL语句在即时查询中也可以查到满足条件数据，但是历史告警中没有产生事件，请问怎么排查呢?
<img width="1359" alt="image" src="https://user-images.githubusercontent.com/40685086/195066242-739e319c-ada8-402a-983e-5ae19c25824c.png">
<img width="1585" alt="image" src="https://user-images.githubusercontent.com/40685086/195066387-5e314a34-2edc-4240-b144-b61b7a6d280a.png">
<img width="1786" alt="image" src="https://user-images.githubusercontent.com/40685086/195066465-35013add-6c3b-4d1e-990e-acec409bfc16.png">

**Comment by UlricQin** [comment]
夜莺v5目前的架构，是一个时序库对应一个n9e-server，n9e-server的配置文件server.conf中除了要配置Reader的地址（就是TSDB的地址），还要特别注意配置ClusterName，你检查一下你的DevCluster这个集群对应的n9e-server的配置中ClusterName是叫DevCluster么

**Comment by ghost** [comment]
n9e-server绑定独立的DevCluster集群，n9e-server的配置文件server.conf中Reader和ClusterName都有配置，历史告警中还是没有产生事件,但是默认的集群里可以收到告警事件。

**Comment by UlricQin** [comment]
你有两个时序库，一个Default，一个DevCluster，也部署了两个 n9e-server，一个n9e-server处理Default时序库，一个n9e-server处理DevCluster？如果是这样的话理论上是应该告警的。

另外升级到最新版本试试，如果是老版本可能有问题
